content/id/docs/tasks/tools/install-minikube.md

```diff
@@ -63,9 +63,9 @@ Hyper-V Requirements:     A hypervisor has been detected. Features required for
 {{< tabs name="tab_with_md" >}}
 {{% tab name="Linux" %}}
 
-### Menginstal kubectl
+### Instalasi kubectl
 
-Pastikan kamu mempunyai kubectl yang terinstal. Kamu bisa menginstal kubectl dengan mengikuti instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/id/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux).
+Pastikan kamu mempunyai kubectl yang terinstal. Kamu bisa menginstal kubectl dengan mengikuti instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/id/docs/tasks/tools/install-kubectl/#menginstal-kubectl-pada-linux).
```

Comment:
```suggestion
Pastikan kamu sudah menginstal kubectl. Kamu bisa menginstal kubectl dengan mengikuti instruksi pada halaman [Menginstal dan Menyiapkan kubectl](/id/docs/tasks/tools/install-kubectl/#menginstal-kubectl-pada-linux).
```

Reply:
I've changed it. You can check it again :D